ImprovedAnvil/lib/dialogues/02_decompile_and_patch.tph

```diff
@@ -132,4 +132,22 @@ COPY_EXISTING ~NEERAJ.DLG~ ~override~
 COPY_EXISTING ~OHNTELAN.DLG~ ~override~
  DECOMPILE_AND_PATCH BEGIN
    REPLACE_TEXTUALLY ~10000~ ~2000~
- END
\ No newline at end of file
+ END
+ 
+COPY_EXISTING ~BEDWIN.DLG~ ~override~
+ DECOMPILE_AND_PATCH BEGIN
+   REPLACE_TEXTUALLY ~HasItem("ohhcloak","Hexxat")~ ~OR(3) HasItem("ohhcloak","Hexxat") HasItem("S!clck08","Hexxat") HasItem("S!clck09","Hexxat")~
+ END
+
+COPY_EXISTING ~BHEXXA25.DLG~ ~override~
+ DECOMPILE_AND_PATCH BEGIN
+   REPLACE_TEXTUALLY ~HasItem("ohhcloak",Myself)~ ~OR(3) HasItem("ohhcloak",Myself) HasItem("S!clck08",Myself) HasItem("S!clck09",Myself)~
+ END
+
+COPY_EXISTING ~OHHPHREY.DLG~ ~override~
+ DECOMPILE_AND_PATCH BEGIN
+   REPLACE_TEXTUALLY ~!HasItemEquipedReal("ohhcloak","hexxat")~ ~tempvar1~
+   REPLACE_TEXTUALLY ~HasItemEquipedReal("ohhcloak","hexxat")~ ~OR(3) HasItemEquipedReal("ohhcloak","hexxat") HasItemEquipedReal("S!clck08","hexxat") HasItemEquipedReal("S!clck09","hexxat")~
+   REPLACE_TEXTUALLY ~tempvar1~ ~!HasItemEquipedReal("ohhcloak","hexxat") !HasItemEquipedReal("S!clck08","hexxat") !HasItemEquipedReal("S!clck09","hexxat")~
+   REPLACE_TEXTUALLY ~ActionOverride("hexxat",GiveItem("ohhcloak","ohhphrey"))~ ~ActionOverride("hexxat",GiveItem("ohhcloak","ohhphrey")) ActionOverride("hexxat",GiveItem("S!clck08","ohhphrey")) ActionOverride("hexxat",GiveItem("S!clck09","ohhphrey"))~
```

Comment:
I wonder what happens to GiveItem() if the item does not exist. Hopefully, just nothing and the scripts won't break